src/plugins/console/server/lib/spec_definitions/json/generated/nodes.stats.json

```diff
@@ -59,6 +59,6 @@
         "warmer"
       ]
     },
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/cluster-nodes-stats.html"
+    "documentation": "https://opensearch.org/docs/latest/guide/en/elasticsearch/reference/master/cluster-nodes-stats.html"
```

Comment:
I think the API is listed here but it doesn't really have any details: https://opensearch.org/docs/1.2/opensearch/popular-api/#get-node-statistics